app/js/event/event.filter.js

```diff
@@ -0,0 +1,23 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('upcomingStops.event')
+    .filter('distanceFormat', distanceFormat);
+
+  distanceFormat.$inject = [];
+
+  function distanceFormat() {
+
+    return function(x) {
+      var distance;
+      if(isNaN(x)){
+        return;
```

Comment:
The filter should return the original value if it is not a number.
In other words, line 15 should be
```
        return x;
```